common/crypto_utils/avalon_crypto_utils/crypto_utility.py

```diff
@@ -253,6 +255,11 @@ def byte_array_to_hex(byte_array):
     return hex_value.upper()
 
 
+# -----------------------------------------------------------------------------
+def hex_to_byte_array(hex_str):
```

Comment:
This seems like a duplicate of the hex_utils method. What is the reason for this?

If you keep it, add a docstring.  Adapted from hex_string.h:

```
     """
     Convert a hex string (i.e., a string of characters with values      
     between '0'-'9', 'A'-'F') to an array of bytes.
     """
```

Reply:
Since it's just a wrapper, better use the hex_util directly. Why have this specifically? Even if wrapping is needed, put that flavour in the hex_util as well.

Reply:
Will delete the wrapper function and use the one in hex_utils.py

Reply:
Deleted duplicate wrapper function and used the function in hex_uils.py